packages/auth/src/accessToken/service.ts

```diff
@@ -93,7 +94,16 @@ async function introspect(
       return { active: false }
     }
     const key = await deps.clientService.getKeyByKid(grant.clientKeyId)
-    return { active: true, ...grant, key: { proof: 'httpsig', jwk: key } }
+    const clientId = crypto
+      .createHash('sha256')
+      .update(key.client.id)
```

Comment:
Should the `id` inside of `key.client` be removed from the response, or is obscuring the client's id not the intent?

Reply:
Actually, I think `jwk` shouldn't include the entire `client` object. The spec doesn't have it. I'm surprise the validation didn't pick it up.